themes/digital.gov/layouts/partials/core/head.html

```diff
@@ -206,9 +206,6 @@
   {{- if $.Params.source_url -}}
     {{- "<!-- Redirect =============== -->" | safeHTML }}
     <meta http-equiv="refresh" content="0; url={{- $.Params.source_url -}}" />
-  <script type="text/javascript">
-    window.location.href = "{{- $.Params.source_url -}}?dg"
-  </script>
```

Comment:
This took the source URL and appended `?dg` at the end. Now that it's gone it looks like there's some inconsistency between `$.Params.redirectto` and `$.Params.source_url` since they both redirect.

Can you create a card in the backlog to investigate what they're doing and confirm if we need both of them?

Reply:
ticket has been created to look into this and address any changes